docs/troubleshooting.md

```diff
@@ -22,3 +22,4 @@ discusses potential solutions to the same.
 | Repository changed 'Origin' and 'Label' error while running `apt-get update` command:  <br/>`E: Repository 'http://packages.cloud.google.com/apt gcsfuse-focal InRelease' changed its 'Origin' value from 'gcsfuse-jessie' to 'namespaces/gcs-fuse-prod/repositories/gcsfuse-focal'`<br/>`E: Repository 'http://packages.cloud.google.com/apt gcsfuse-focal InRelease' changed its 'Label' value from 'gcsfuse-jessie' to 'namespaces/gcs-fuse-prod/repositories/gcsfuse-focal'`<br/>`N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details. ` | Use one of the following commands to upgrade to latest GCSFuse version<br/> `sudo apt-get update --allow-releaseinfo-change `<br/>OR<br/>`sudo apt update -y && sudo apt-get update`                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | Unable to unmount or stop GCSFuse due to an error message like:`fusermount: failed to unmount: Device or resource busy` or `umount: /path/to/mountpoint: target is busy`.</br>This typically indicates active processes are using files or directories within the GCSFuse mount.                                                                                                                                                                                                                                                                                                                                      | Find the process ID of GCSFuse:<br/>`BUCKET=<Enter your bucket name>`</br>` MOUNT_POINT=<Enter your mount point>`</br>`PID=$(ps -aux &#124; grep "gcsfuse.*$BUCKET.*$MOUNT_POINT" &#124; grep -v grep &#124; tr -s ' ' &#124; cut -d' ' -f2)`</br>Kill the GCSFuse process:</br>`sudo kill -SIGKILL "$PID"`</br>Unmount GCSFuse</br>`fusermount -u $"MOUNT_POINT"`                                                                                                                                                                                                                                                     |
+|ls: reading directory ...:  Input/output error|Find out if you have any object(s) having name/prefix having `//` in it or start with `/` in the mounted GCS bucket. If yes, move, rename or delete it|
```

Comment:
Please add a link to the Semantics doc link

Reply:
Good point. Let's finalize the heading there, then add I'll address this comment, or I'll have to redo this work.

Reply:
Added.